### DISCUSSION TITLE
support for reSTructuredText (👍12)

### DISCUSSION THREAD

**Discussion by charlyoleg** [11ty/eleventy#4030] [Enhancement Queue]
reStructuredText (*.rst) is another popular mardown format, probably because of Sphinx and readthedocs.com. It would be great that 11ty suppoorts *.rst beside *.md

**Comment by zachleat** [comment]
I think this will be a user-land thing, see #117. But I’ll put into the queue!

**Comment by zachleat** [comment]
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

**Comment by zachleat** [comment]
Home base for this one is over at #3981
